Drive/rover_drive_system.hpp

```diff
@@ -22,10 +22,12 @@ class RoverDriveSystem
   /// Mission controls possible input values
   struct MissionControlData
   {
-    bool is_operational;
+    int is_operational;
```

Comment:
Should behave the same way. I did this so that JSON parser wouldn't get mad at me for using %d to assign a boolean value.

Reply:
Yeah this is fine. You'll need mission control to send you an into though, rather than a "true" or "false"